modules/Microsoft.Network/applicationGateways/.test/.scripts/New-Certificate.ps1

```diff
@@ -0,0 +1,40 @@
+﻿param(
```

Comment:
Adding a point to issue #2038 to improve deployment scripts (e.g. add synopsis). Outside of this PR.